tests/X86/Run.cpp

```diff
@@ -458,6 +490,14 @@ static void RunWithFlags(const test::TestInfo *info,
   memset(lifted_state->x87.fxsave.st, 0, kill_size);
   memset(native_state->x87.fxsave.st, 0, kill_size);
 
+#if 32 == ADDRESS_SIZE_BITS
```

Comment:
I meant to remove the `#if` and just depend on the actual `if` :-P

Reply:
Oh, that makes more sense (-:

The reason I used `#if` is because the referenced fields don't exist in 64-bit mode, so you get a compiler error.

Reply:
Oh duh! I didn't make that connection.. lol. So then the additional check in the `if` is unnecessary.

Reply:
Apparently not? One moment...

Reply:
Nope, yes, the `#if` is necessary, I just assumed you were talking about some other warning I missed. Reverting the redundant expression.